cli_FSD/api_routes.py

```
@@ -15,35 +15,117 @@
 
+def extract_keywords(query):
+    """Extract meaningful keywords from natural language query"""
+    import re
+    
+    # Remove common question words and phrases
+    stop_words = {
+        'what', 'do', 'you', 'know', 'about', 'tell', 'me', 'explain', 
+        'describe', 'how', 'why', 'when', 'where', 'can', 'could', 
+        'would', 'should', 'is', 'are', 'was', 'were', 'the', 'a', 
+        'an', 'and', 'or', 'but', 'in', 'on', 'at', 'to', 'for',
+        'of', 'with', 'by', 'from', 'up', 'about', 'into', 'through',
```

Ruff (0.11.9)

Sets should not contain duplicate item `'about'`

Remove duplicate item

(B033)